app/javascript/components/collections/landing/SearchResultsCard.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CardMetaData = ({ doc, fieldLabel, fieldName }) => {
+  if (doc[fieldName]) {
+    return (
+      <>
+        <dt>{fieldLabel}</dt>
+        {Array.isArray(doc[fieldName]) && doc[fieldName].length > 1 ? (
+          <dd>{doc[fieldName].join(', ')}</dd>
+        ) : (
+          <dd>{doc[fieldName]}</dd>
+        )}
+      </>
+    );
+  }
+  return null;
+};
+
+CardMetaData.propTypes = {
+  doc: PropTypes.object,
+  fieldLabel: PropTypes.string,
+  fieldName: PropTypes.string
+};
+
+const millisecondsToFormattedTime = ms => {
+  let sec_num = ms / 1000;
+  let hours = Math.floor(sec_num / 3600);
+  let minutes = Math.floor((sec_num % 3600) / 60);
+  let seconds = sec_num - minutes * 60 - hours * 3600;
+
+  let hourStr = hours < 10 ? `0${hours}` : `${hours}`;
+  let minStr = minutes < 10 ? `0${minutes}` : `${minutes}`;
+  let secStr = seconds.toFixed(0);
+  secStr = seconds < 10 ? `0${secStr}` : `${secStr}`;
+
+  return `${hourStr}:${minStr}:${secStr}`;
```

Comment:
Maybe this is a good opportunity to pull in the `moment` package to  handle date / time parsing?

Reply:
Since this function works properly I think this is better than pulling in an external package for only this. If we have any other places where we can use it, then I think it's a good idea to pull it in. What do you think?
Maybe we can refactor this to a smaller code block?

Reply:
@Dananji Agreed.  Let's keep it easy for now and maybe just keep in mind for the future.  Good call.